EasyPost.EasyVCR/AdvancedSettings.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using EasyPost.EasyVCR.Interfaces;
+using EasyPost.EasyVCR.InternalUtilities;
+
+namespace EasyPost.EasyVCR
+{
+    /// <summary>
+    ///     Advanced settings for EasyVCR.
+    /// </summary>
+    public class AdvancedSettings
+    {
+        /// <summary>
+        ///     Rules to use when evaluating recordings.
+        /// </summary>
+        public MatchRules? MatchRules = MatchRules.Default;
+
+        /// <summary>
+        ///     Censors to use when building requests and responses.
+        /// </summary>
+        public Censors? Censors = Censors.Default;
+
+        /// <summary>
+        ///     Override how HttpRequestMessage and HttpResponseMessage objects are converted to Request and Response objects.
+        ///     Not recommended to use unless System.Net.Http has introduced breaking changes.
+        /// </summary>
+        public IInteractionConverter? InteractionConverter = new DefaultInteractionConverter();
+
+        /// <summary>
+        ///     Simulate the original request's duration. Overrides <see cref="ManualDelay"/> if set.
+        /// </summary>
+        public bool SimulateDelay = false;
+
+        /// <summary>
+        ///     Simulate a delay in milliseconds for the request.
```

Comment:
In case people want to make their HTTP playback as realistic as possible, testing timing, etc.
We probably won't use it for our immediate purposes (and it's off by default), but just a nice addition to have.